app/views/tags/edit/_organisations.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render "tags/edit/multi_tag",
```

Comment:
This view (and similar) should have their filename suffixed with `_field` to be consistent with the content fields.